### PR TITLE
More incbin work

### PR DESCRIPTION
**Pull request by ProjectRevoTPP**

### CODE REVIEW

File: data/text/credits.s [hunk 1]
```diff
@@ -279,140 +279,678 @@ Text_Version:
...
+
+CreditsABA::
 	credits_entry 0,  Text_ABA
+
+CreditsRNZFotB::
 	credits_entry 0,  Text_RNZFotB
+	
+@ pointer table to credits entries
+
+	.global gCreditsEntryPointerTable
```
Comment (YamaArashi) [inline data/text/credits.s]: Could you remove this ".global" and add an ".align 2"?

File: data/data2.s [hunk 2]
```diff
@@ -2690,10 +2690,7 @@ gUnknown_082162F8:: @ 82162F8
 	.incbin "baserom.gba", 0x002162f8, 0x8
 
 gUnknown_08216300:: @ 8216300
-	.incbin "baserom.gba", 0x00216300, 0x3
-
-gUnknown_08216303:: @ 8216303
```
Comment (YamaArashi) [inline data/data2.s]: Why should this incbin be combined with the previous one? It's not clear they really belong together.

File: data/data2.s [hunk 3]
```diff
@@ -23910,7 +23839,18 @@ gUnknown_0840B258:: @ 840B258
 	obj_tiles 0x08d1a7f4, 256, 0xd6f7
 
 gUnknown_0840B2B8:: @ 840B2B8
```
Comment (YamaArashi) [inline data/data2.s]: This needs an ".align 2".

File: data/data2.s [hunk 4]
```diff
@@ -24914,7 +24851,6 @@ ContestStatsText_VeryHard:
 	.string "Very hard$"
 ContestStatsText_SuperHard:
 	.string "Super hard$"
-
```
Comment (YamaArashi) [inline data/data2.s]: I put this line break to separate the groups of strings.

**Comment by ProjectRevoTPP** [comment]
calls to the beginning of those 8 bytes:
```
_0804EFC8: .4byte gUnknown_082162F8
_0804EFCC: .4byte gUnknown_08216300

_0804F614: .4byte gUnknown_082162F8
_0804F618: .4byte gUnknown_08216300

_0804F6E8: .4byte gUnknown_082162F8
_0804F6EC: .4byte gUnknown_08216300

_0804EEF4: .4byte gUnknown_082162F8
_0804EEF8: .4byte gUnknown_08216300
```

calls to 3 bytes after:

```	subs r0, 0x1C
	cmp r2, r0
	bcc _0804F1B0
	adds r0, 0x8
	cmp r2, r0
	bcs _0804F1B0
	movs r0, 0x2
	b _0804F1B6
	.align 2, 0
_0804F1A8: .4byte 0x03004854
_0804F1AC: .4byte gUnknown_08216300 + 0x3
_0804F1B0:
	movs r0, 0x1
	b _0804F1B6
_0804F1B4:
	movs r0, 0```

```_0804FA78: .4byte 0x03004854
_0804FA7C: .4byte 0x0000ffff
_0804FA80: .4byte gUnknown_08216300 + 0x3
_0804FA84: .4byte 0x03004b20
_0804FA88: .4byte 0x0000014b
_0804FA8C: .4byte 0x0000028f
_0804FA90: .4byte 0x000001f3
_0804FA94: .4byte 0x03002a20
_0804FA98: .4byte 0x00004523
_0804FA9C:
	ldr r1, _0804FAA4
	ldr r0, _0804FAA8
	strh r0, [r1, 0x14]
	b _0804FAD4
	.align 2, 0

```_0804FB9C: .4byte 0x03004854
_0804FBA0: .4byte 0x0000ffff
_0804FBA4: .4byte gUnknown_08216300 + 0x3
_0804FBA8: .4byte 0x03004b20
_0804FBAC: .4byte 0x0000014b
_0804FBB0: .4byte 0x0000028f
_0804FBB4: .4byte 0x000001f3
_0804FBB8: .4byte 0x03002a20
_0804FBBC: .4byte 0x00004523
_0804FBC0:
	ldr r1, _0804FBC8
	ldr r0, _0804FBCC
	strh r0, [r1, 0x16]
	b _0804FC02
	.align 2, 0
_0804FBC8: .4byte 0x03002a20
_0804FBCC: .4byte 0x00005432
_0804FBD0:
	cmp r2, 0x3C
	bls _0804FBE4
	ldr r1, _0804FBDC
	ldr r0, _0804FBE0
	b _0804FBF4
	.align 2, 0```

when combined the section of data2 looks like:

```gUnknown_082162EC:: @ 82162EC
	.incbin "baserom.gba", 0x002162ec, 0xc

gUnknown_082162F8:: @ 82162F8
	.incbin "baserom.gba", 0x002162f8, 0x8

gUnknown_08216300:: @ 8216300
	.incbin "baserom.gba", 0x00216300, 0x8

gUnknown_08216308:: @ 8216308
	.incbin "baserom.gba", 0x00216308, 0xc```

making it consistent with the referenced lumps around this. I made the call that the two should be combined.

**Comment by YamaArashi** [comment]
They seem to be used differently. I don't see why the sizes of adjacent incbins would have to be consistent, so I am not sure why that would require they be merged.

**Pull request merged**